react-native/android/app/proguard-rules.pro

```diff
@@ -15,3 +15,46 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 #   public *;
 #}
```

Comment:
this all came from react-native's upgrade config, @MarcoPolo please sanity check this